### PR TITLE
DM-17968: Update deployment URL to lsst-lsp-stable

### PR DESCRIPTION
**Pull request by jonathansick**
Draft: https://nb.lsst.io/v/DM-17968/

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ nb.lsst.io documentation
 ########################
 
 This is the (prototype) documentation for the notebook aspect of the LSST Science Platform.
-This site supports our current users of https://lsst-lspdev.ncsa.illinois.edu/nb and is proof-of-concept for documenting the LSST Science Platform as a whole.
+This site supports our current users of https://lsst-lsp-stable.ncsa.illinois.edu/ and is a proof-of-concept for documenting the LSST Science Platform as a whole.
```
Comment (SimonKrughoff) [inline README.rst]: Here and below, we still need the `/nb` extension in order to go directly to the notebook login page.

Comment (SimonKrughoff) [reply README.rst]: Given that this is documentation specifically for the notebook aspect, I'd advocate for keeping `nb` in the exposed URLs.

File: conf.py [hunk 2]
```diff
@@ -83,7 +83,7 @@
 
 # -- Options for linkcheck builder ----------------------------------------
 
-linkcheck_ignore = [r'^https://lsst-lspdev\.ncsa\.illinois\.edu',
+linkcheck_ignore = [r'^https://lsst-lsp-stable\.ncsa\.illinois\.edu',
```
Comment (SimonKrughoff) [inline conf.py:88]: I'm not exactly sure what this is doing, but there is now a landing page at that location.

File: getting-started/logging-in.rst [hunk 3]
```diff
@@ -44,7 +44,7 @@ If you've forgotten your password, you can reset it by visiting https://identity
 Step 2: Log in
 ==============
 
-Open `lsst-lspdev.ncsa.illinois.edu/nb <https://lsst-lspdev.ncsa.illinois.edu/nb>`__ in your web browser.
+Open `lsst-lsp-stable.ncsa.illinois.edu <https://lsst-lsp-stable.ncsa.illinois.edu/>`__ in your web browser.
```
Comment (SimonKrughoff) [inline getting-started/logging-in.rst]: As above, keep `/nb`, I think.

File: index.rst [hunk 4]
```diff
@@ -9,7 +9,7 @@ LSST Science Platform Notebook Aspect Documentation
 The Notebook Aspect enables you to do your science at the LSST Data Facility, with the LSST Science Pipelines, a full suite of development tools, and your own Python code.
 The Notebook Aspect is powered by the JupyterLab_ project.
 
-This documentation is specific to the deployment at https://lsst-lspdev.ncsa.illinois.edu/nb.
+This documentation is specific to the deployment at https://lsst-lsp-stable.ncsa.illinois.edu/.
```
Comment (SimonKrughoff) [inline index.rst]: And here as well (`nb`).

**Review by SimonKrughoff — APPROVED** [review]
As I mention, I think we should keep `nb` on the end of exposed URLs.

**Comment by jonathansick** [comment]
Good ideas, thanks.

**Pull request merged**